lib/clerk/railtie.rb

```diff
@@ -5,6 +5,12 @@ module Clerk
   class Railtie < Rails::Railtie
     initializer 'clerk.ar_extensions' do |app|
       ActiveRecord::Base.extend Clerk
+
+      #initialize configuration object
+      Clerk.configure do |config|
+        config.silence_warnings = Rails.env.test?
```

Comment:
Yeah, I was originally not going to default to true in test env, but as I was writing the docs, it make it seem like you needed another step to get it to work how you expect it to work (aka, no warning messages in your tests).

Can you send me the rubocop.yml file and I can fix it based on that (and maybe add it to the project as well 😜)